mozilla_django_oidc_db/models.py

```diff
@@ -13,6 +15,13 @@
 
 import mozilla_django_oidc_db.settings as oidc_settings
 
+# `classproperty` was moved to another module in Django 3.1
+# See https://github.com/django/django/blob/ca9872905559026af82000e46cde6f7dedc897b6/docs/releases/3.1.txt#L649
+if StrictVersion(django.__version__) >= StrictVersion("3.1"):
+    from django.utils.functional import classproperty
+else:
+    from django.utils.decorators import classproperty
```

Comment:
the preferred approach is to have a `compat.py` module which shims these imports & rather catch the `ImportError` than comparing version numbers:

```py
try:
    from django.utils.functional import classproperty
except ImportError:
    from django.utils.decorators import classproperty
```
then in your application code you can just call:

```py
from .compat import classproperty
```